assets/pages/subscriptions-landing/components/featuredProducts.jsx

```diff
@@ -42,10 +44,28 @@ const dpImage = (
     ]}
     fallback="digitalPackFlashSaleDesktop"
     fallbackSize={500}
-    altText="ad-free, live and discover, and the daily edition"
+    altText=""
     fallbackImgType="png"
   />);
 
+const weeklyImage = (
+  <GridImage
+    gridId="guardianWeeklyHero"
+    srcSizes={[140, 500, 1000, 1080]}
```

Comment:
I'm not sure that extra 80px is worth it.

Additionally, can we use check Grid has created crops of these widths? For more context see [here](https://github.com/guardian/support-frontend/pull/1107), the patron image didn't have a 1000px wide crop as the source image was ~800px.

For reference, the standard Grid widths are defined [here](https://github.com/guardian/grid/blob/master/cropper/app/lib/CropperConfig.scala#L29-L30).

Reply:
it totally isn't! it exists (that's the original image) but yeah it's not necessary